apps/supernova/src/components/filters/FilterSelect.jsx

```diff
@@ -84,13 +84,17 @@ const FilterSelect = () => {
           value={filterLabel}
           onChange={(val) => handleFilterLabelChange(val)}
         >
-          {filterLabels?.map((filter) => (
-            <SelectOption
-              value={filter}
-              label={humanizeString(filter)}
-              key={filter}
-            />
-          ))}
+          {filterLabels &&
+            [].slice
```

Comment:
Why not just do `filterLabels?.sort().map((filter)) => ...` ?
What advantage does the `[].slice.call(filterLabels)` give?